configure.ac

```diff
@@ -314,9 +314,23 @@ dnl include all subsystem m4 fragments now that the core autoconf functionality
 dnl has been setup.  No fragment should do anything except define
 dnl PAC_SUBCFG_{PREREQ,BODY} macros which will be expanded later as
 dnl appropriate
-# begin subsys includes
-m4_include([subsys_include.m4])
-# end subsys includes
+m4_include([src/mpid/ch4/subconfigure.m4])
+m4_include([src/binding/fortran/use_mpi/subconfigure.m4])
+m4_include([src/pmi/subconfigure.m4])
+m4_include([src/mpid/common/subconfigure.m4])
+m4_include([src/util/logging/subconfigure.m4])
+m4_include([src/mpid/ch3/subconfigure.m4])
+m4_include([src/pm/subconfigure.m4])
```

Comment:
I was hoping that this can be much more hierarchical.  Basically equivalent to `Makefile.mk` at each directory-level, more-or-less.

Reply:
Same comment for the other instances of `m4_include` too.

Reply:
hmm, ok that will be much finer grain then. Sure I will fix it.